packages/core/phase2/lib/getOperationSequence/areAllResultsAlreadyPresentOnPnc/isMatchToPncAdjudicationAndDisposals/areDisposalsMatching.ts

```diff
@@ -0,0 +1,15 @@
+import type { PncDisposal } from "../../../../../types/PncQueryResult"
+
+const areStringsEqual = (firstObject: string | undefined, secondObject: string | undefined) =>
+  (!firstObject && !secondObject) || firstObject === secondObject
+
+const areDisposalsMatching = (firstDisposal: PncDisposal, secondDisposal: PncDisposal): boolean =>
```

Comment:
Sorry I should have updated my comment as I added to it, but I think we should have this as `arePncDisposalsMatching` so we're always saying `PncDisposal`?

Reply:
Sorry missed that. Updated